daq_macros.py

```diff
@@ -259,11 +259,87 @@ def run_robot_recovery_procedure():
   """ Generic recovery procedure to be used during automated
   collection"""
   # Recover robot
+  logging.info("Running robot recovery procedure (nuclear option)")
```

Comment:
logger vs. logging?

Reply:
Good catch, fixed in https://github.com/NSLS2/lsdc/pull/399/commits/016572acc673dfedca454f166725235945b69d11